pkg/cmd/release-payload-controller/job_state_controller.go

```diff
@@ -0,0 +1,215 @@
+package release_payload_controller
+
+import (
+	"context"
+	"fmt"
+	"github.com/openshift/release-controller/pkg/apis/release/v1alpha1"
+	releasepayloadclient "github.com/openshift/release-controller/pkg/client/clientset/versioned/typed/release/v1alpha1"
+	releasepayloadinformer "github.com/openshift/release-controller/pkg/client/informers/externalversions/release/v1alpha1"
+	"github.com/openshift/release-controller/pkg/releasepayload/jobstatus"
+	releasepayloadhelpers "github.com/openshift/release-controller/pkg/releasepayload/v1alpha1helpers"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/util/workqueue"
+	"k8s.io/klog/v2"
+	"reflect"
+
+	"github.com/openshift/library-go/pkg/operator/events"
+)
+
+const (
+	analysisSuccessThreshold = 0.6
```

Comment:
requires doc.

Reply:
wait, one call site?   move it down near usage

Reply:
Done!  (although it pains me :smile: )